### ISSUE TITLE
Upload of app - Manifest parsing has failed

### ISSUE THREAD

**Issue by joni-mcglade-trg** [OfficeDev/microsoft-teams-icebreaker-app#324]
Hi,

I've ran through the process, however when I try to upload the app to teams I get:

Manifest parsing has failed.

I've tried via my own teams client & via https://admin.teams.microsoft.com.

Thanks,
Joni.

**Comment by joni-mcglade-trg** [comment]
Seems there's all kinds of wrong with this, but if you use this it'll tell you what you need to do: https://dev.teams.microsoft.com/appvalidation.html

Issues with the content of the zip & the size of the images for me!

**Comment by williamaherman** [comment]
did you guys get the bot to start working/responding?

**Comment by joni-mcglade-trg** [comment]
@williamaherman No, I couldn't get it to talk to the services at all & ended up giving up.

**Comment by franklinis** [comment]
Hello Everyone,

The app is no longer updated. I assume the problem you have encountered is because the code in Icebreaker try to use older version json schema which is not apt for new Teams.

Try to modify the second and third line of manifest.json like below and push it back to the Teams as an update or new teams.

```
"$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.17/MicrosoftTeams.schema.json",

"manifestVersion": "1.17",
```

Please let me know how it goes.
